src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/401Tests.kt

```diff
@@ -0,0 +1,58 @@
+package org.vaccineimpact.orderlyweb.tests.unit_tests.templates
+
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.*
+import org.junit.Rule
+import org.junit.Test
+import org.vaccineimpact.orderlyweb.app_start.WebErrorHandler
+import org.vaccineimpact.orderlyweb.test_helpers.TeamcityTests
+import org.vaccineimpact.orderlyweb.tests.unit_tests.templates.rules.FreemarkerTestRule
+import org.xmlmatchers.XmlMatchers.hasXPath
+
+class _401Tests: TeamcityTests()
+{
+    @get:Rule
+    val template = FreemarkerTestRule("401.ftl")
+
+    @Test
+    fun `renders correctly for Montagu auth`()
+    {
+        val mockModel = mock<WebErrorHandler.ErrorViewModel> {
+            on { appName } doReturn "testApp"
+            on { authProvider } doReturn "montagu"
+        }
+
+        val xmlResponse = template.xmlResponseFor(mockModel)
+
+        assertThat(xmlResponse, hasXPath("//h1/text()", equalToIgnoringWhiteSpace("Login failed")))
```

Comment:
looks like `equalToIgnoringWhiteSpace` is deprecated and should be `equalToCompressingWhiteSpace` instead